app/cdap/components/SourceControlManagement/LocalPipelineListView/PipelineTable.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import {
+  Checkbox,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  TableHead,
+  TableContainer,
+} from '@material-ui/core';
+import { setSelectedPipelines } from '../store/ActionCreator';
+import { ILocalPipeline } from '../types';
+import T from 'i18n-react';
+import StatusButton from 'components/StatusButton';
+import { SUPPORT } from 'components/Replicator/Create/Content/Assessment/TablesAssessment/Mappings/Supported';
+
+const PREFIX = 'features.SourceControlManagement.push.table';
+
+interface ILocalPipelineTableProps {
+  localPipelines: ILocalPipeline[];
+  selectedPipelines: string[];
+  showFailedOnly: boolean;
+}
+
+const TableBox = styled(TableContainer)`
+  box-shadow: 0 1px 2px 0 rgb(0 0 0 / 20%);
+  margin-bottom: 30px;
+  max-height: calc(80vh - 200px);
+`;
+
+const StyledTableCell = styled(TableCell)`
+  font-size: 14px;
+`;
+
+const StyledTableRow = styled(TableRow)`
+  &&.Mui-selected {
+    background-color: rgba(25, 118, 210, 0.08);
```

Comment:
Is there a color in the palette you can use?

Reply:
I've been thinking about this - the current color palette in app/cdap/components/ThemeWrapper/colors.ts imports the scss - i think we should normalize the colors in there and not import the scss - basically move away from scss wherever possible - thoughts @njbriggs